piksi_tools/console/console.py

```diff
@@ -242,12 +254,20 @@ def ext_event_callback(self, sbp_msg, **metadata):
   def _paused_button_fired(self):
     self.console_output.paused = not self.console_output.paused
 
```

Comment:
When the gui's log level filter is changed, we need to change the filter trait on the table view as well.  I couldn't figure out how to have the sub-part of the view re-use the upper level trait.